docs/index.rst

```diff
@@ -0,0 +1,73 @@
+Lumi Language Guide
```

Comment:
I would consider steps for making the index page a bit more brief and welcoming.
For instance:
* Moving the 'hello world' link to an earlier point in the page
* Perhaps add a small 'Getting Started' section which is simply a few installation commands + hello world
* Moving the meaning of the name to a different page